CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## v1.4.1 (2019-8-15)
+
+### Bug Fixes
+
+* Drag now works correctly in Chrome Mobile on Android and Chrome device emulator with touch screens ([#3948](https://github.com/DevExpress/testcafe/issues/3948))
+* Live Mode no longer fails when it restarts tests that use `import` on Node.js v12 ([#4052](https://github.com/DevExpress/testcafe/issues/4052))
```

Comment:
It's probably fine, but the problem is not limited to `import`, it can be reproduced with `require` too. Maybe we should rephrase it.

Reply:
Sure